File: solutions/61-jawad/Age_of_villagers/Age_of_villagers/Age_of_villagers/Shapes/Rectangle.cs
using System;
using System.Drawing;
using System.Collections.Generic;
using System.Text;

namespace Age_of_villagers.Shapes
{
    class Rectangle : IShape
    {
        public void Draw(Graphics g, int x, int y, int height, int width)
        {
            Pen p = new Pen(Color.Red);
            g.DrawLine(p, x, y, x + width, y);
            g.DrawLine(p, x, y, x, y - height);
            g.DrawLine(p, x, y - height, x + width, y - height);
            g.DrawLine(p, x + width, y - height, x + width, y);

        }

        public Rectangle shape()
        {
            throw new NotImplementedException();
        }
    }
}

File: solutions/40-arif-JAVA/AgeOfVillagers/src/shape/Rectangle.java
package shape;

import javafx.scene.canvas.Canvas;
import javafx.scene.canvas.GraphicsContext;

public class Rectangle implements IShape {
    GraphicsContext graphicsContext;
    public Rectangle(GraphicsContext graphicsContext) {
        this.graphicsContext = graphicsContext;
    }

    @Override
    public void draw() {

        graphicsContext.fillRect(20,20,20,20);

    }
}
